nix/wire-server.nix

```diff
@@ -442,6 +442,29 @@ let
     };
   }) cabal-plan;
 
+  sbomqs = pkgs.buildGoModule rec {
```

Comment:
Wouldn't this better reside as dedicated `.nix` file in `nix/pkgs`? :thinking:

Reply:
Good idea. Will do that.